pages/dashboard.py

```diff
@@ -2,19 +2,24 @@
 from pages.base import OSFBasePage, BaseElement
 from selenium.webdriver.common.by import By
 
+
 class DashboardPage(OSFBasePage):
 
     locators = {
-        'create_project_button':(By.CSS_SELECTOR, 'button.btn-success:nth-child(1)', settings.LONG_TIMEOUT),
+        **OSFBasePage.locators,
+        **{
+            'identity': (By.CSS_SELECTOR, 'body > div#osfHome > div.prereg-banner > div.container', settings.LONG_TIMEOUT),
+            'create_project_button': (By.CSS_SELECTOR, 'button.btn-success:nth-child(1)', settings.LONG_TIMEOUT),
+        }
     }
 
-    def __init__(self, driver):
-        super(DashboardPage, self).__init__(driver)
-        if not self.is_logged_in:
+    def __init__(self, driver, goto=True, require_login=True):
+        super(DashboardPage, self).__init__(driver, goto, require_login)
+        if not self.is_logged_in():
```

Comment:
This should always require login `super(DashboardPage, self).__init__(driver, goto, require_login=True)`. And then we can remove the raise ValueError code I have here.